docs/src/styles/style.css

```diff
@@ -0,0 +1,55 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,400;0,500;0,600;0,700;0,800;1,400;1,500;1,600;1,700;1,800&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;1,100;1,300;1,400;1,500;1,700&display=swap");
+@import url("null.css");
+@import url("header.css");
+@import url("footer.css");
+
+.wrapper {
+  display: flex;
+  flex-direction: column;
+  margin: 0 auto;
+  min-width: 400px;
+  min-height: 100vh;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  max-width: 930px;
+  margin: 0 auto;
+}
+
+.main {
+  flex-grow: 1;
+}
+
+.main__title {
+  font-family: Roboto sans-serif;
```

Comment:
Something wrong with font connection - please don't forget to add safe font and tune up connection of Roboto properly